near-plugins/tests/contracts/pausable/Makefile

```diff
@@ -0,0 +1,6 @@
+build:
```

Comment:
I think `build` and `expand` should be added as ["phony" targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) to following Makefile conventions.

Reply:
Done in 93887f3.